src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.PlatformServices/Helpers/RequestHelper.cs

```diff
@@ -0,0 +1,37 @@
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace Altinn.App.PlatformServices.Helpers
+{
+    public static class RequestHelper
+    {
+        public static string GetComplientContentHeader(string headerValues)
```

Comment:
I believe the correct spelling is `compliant`.

Reply:
fixed